src/jit/gentree.h

```diff
@@ -2399,12 +2399,23 @@ struct ReturnTypeDesc
 public:
     ReturnTypeDesc()
     {
+#ifdef _TARGET_X86_
+        m_regType0 = TYP_INT;
+        m_regType1 = TYP_INT;
+
+#ifdef DEBUG
+        m_inited = true;
+#endif
+
+#else
```

Comment:
@adiaaida - even if we initialize ReturnTypeDesc in default constructor, still it will not solve the issue of fgMorphIntoHelperCall(). In that particular case, an existing gentree node was getting morph'ed into a GT_CALL node by changing its oper and other fields.